packages/create-sitecore-jss/src/templates/nextjs/next.config.js

```diff
@@ -29,6 +29,14 @@ const nextConfig = {
   // Enable React Strict Mode
   reactStrictMode: true,
 
+  // enable image provider domains and device sizes
+  images: {
+    domains: ['cm.jss.localhost'],
+    loader: 'custom',
+    path: 'https://cm.jss.localhost',
```

Comment:
Same as above

Reply:
Do we need `loader` and `path` config here? I thought they didn't end up working as we were hoping with our default loader code.

Reply:
No, we don't they don't seem to work with the 'custom' loader.